utils/remote.c

```diff
@@ -645,6 +647,22 @@ static void addDays(tr_variant* args, tr_quark const key, char const* arg)
     }
 }
 
+static void addLabels(tr_variant* args, const tr_quark key, const char* arg)
```

Comment:
There is no point in passing key if the function is called `addLabels` (i.e. we already know what the key is). Either remove the parameter or rename the function to be more generic.